server/src/__tests__/item.test.ts

```diff
@@ -0,0 +1,201 @@
+// eslint-disable-next-line import/no-extraneous-dependencies
+import request from "supertest";
+import connection from "../db/connection";
+import app from "../app";
+
+const HOME_ROUTE = "/items";
+let itemId: string;
+const INVALID_UUID = "thisisan-inva-lidu-uidv-aluesoerror";
+const NON_EXISTENT_UUID = "12345678-1234-1234-1234-1234567890AB";
+let supplierId: string;
+
+describe("Operation on the item route", () => {
```

Comment:
There are no tests validating data when user tries to update an item, there should be tests for that. Eg:
- "Should return 400 if user sends incorrect data"
- "should return 200 if user sends correct data"

Reply:
I have test cases for these two points on line 21 & line 86 respectivly.